scripts/entry.ps1

```diff
@@ -13,8 +13,8 @@ $SCRIPT_PATH = ("{0}\{1}.ps1" -f $PSScriptRoot, $Args[0])
 if (Test-Path $SCRIPT_PATH -ErrorAction Ignore)
 {
     Invoke-Expression -Command $SCRIPT_PATH
-    if (-not $?) {
-        exit 255
+    if ($LASTEXITCODE -ne 0) {
```

Comment:
Required so that Docker build failures actually cause a failure on PR workflows; before, these errors were squashed and never returned.